src/cc_print.c

```diff
@@ -47,6 +47,22 @@ cc_print_uint64_unsafe(char *buf, uint64_t n)
 }
 
 size_t
+cc_print_int64_unsafe(char *buf, int64_t n)
+{
+    size_t d;
+    uint64_t ab = llabs(n);
```

Comment:
apparently `llabs` returns an `int64_t` and if `n` is `INT64_MIN` the behaviour is undefined... do we want to support that case? http://www.gnu.org/software/libc/manual/html_node/Absolute-Value.html

Some thing happens in the non-unsafe function.